cmake/ComputeCppConfig.cmake

```diff
@@ -1,5 +1,5 @@
 #.rst:
-# FindComputeCpp
+# ComputeCppConfig
```

Comment:
I would recommend against adding this file to the repository. The idea is that ComputeCpp distributes the Config file, such that any user who downloads ComputeCpp can specify `-DComputeCpp_DIR=/path/to/computecpp/lib/cmake` (or some similar path) to use it directly from the package, without needing the code in the repository. (Besides, even if you do want to have this file in-repo, the one in the package is slightly out-of-date for the moment. The changes in this PR are effectively going *back* in time.)

Reply:
Good point @DuncanMcBain - let me test that locally & make the change